File: TOI-Zero_68/A2/008/008.py
n = int(input())
cars = []

for i in range(n):
    x, y = map(int, input().split())
    cars.append((x, y))

cars.sort()  # Sort by x-coordinate

count = 0
for i in range(1, n):
    if cars[i][1] < cars[i-1][1]:
        count += 1
    cars[i] = (cars[i][0], max(cars[i][1], cars[i-1][1]))

print(count) 

File: TOI-Zero_68/A2/008/008.cpp
#include<bits/stdc++.h>
using namespace std;

#define ll long long

int main(){
	ios::sync_with_stdio(0);
	cin.tie(0);
	int n;
    cin >> n;
    pair<int, int> car[n];
    for (int i=0;i<n;i++) {
        cin >> car[i].first >> car[i].second;
    }
    sort(car,car+n);
	int count=0;
    for (int i=1;i<n;i++) {
        count += (car[i].second<car[i-1].second);
        car[i].second=max(car[i].second,car[i-1].second);
    }
    cout << count;
}
